js/filter-control.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var $ = require('jquery');
+
+function FilterControl($selector) {
+  this.$body = $selector;
+  this.modifiesFilter = this.$body.data('modifies-filter');
+  this.modifiesProperty = this.$body.data('modifies-property');
+  this.getValue();
+
+  this.controlledFilter = $('#' + this.$body.data('controls'));
+  this.$body.on('change', this.handleChange.bind(this));
+}
+
+FilterControl.prototype.getValue = function() {
+  var values = [];
+  this.$body.find('input:checked').each(function() {
+    values.push($(this).val());
+  });
+  this.values = values;
+};
```

Comment:
Rather than keeping `values` as state on the the `FilterControl`, you can just return `values`. Since you're using event triggering anyway to pass the values, it doesn't need to be kept as state. I prefer to avoid state and state mutation when possible. It avoids a whole class of bugs and debugging headaches, IMHO.